src/otg/v2g.py

```diff
@@ -62,22 +70,24 @@ def run(self: V2GStep) -> None:
             va_slimmed.get_plof_v2g(gene_index_filtered),
             # intervals
             IntervalsAndersson.parse(
-                IntervalsAndersson.read_andersson(self.session, self.anderson_path),
+                IntervalsAndersson.read_andersson(
```

Comment:
I might be just a huge pain, but I feel, the list of the included intervals should not be hardcoded. I think we should have datasources to process interval datasets with consistent schema ready to be ingested by v2g, however, v2g should not be aware of what are the explicit list of the available datasets. This implementation makes it more complicated to introduce more intervals. What do you think?